lib/qiita/markdown/processor.rb

```diff
@@ -15,6 +15,7 @@ class Processor
         Filters::SyntaxHighlight,
         Filters::Mention,
         Filters::Sanitize,
+        Filters::ExternalLink,
```

Comment:
The `Sanitize` filter should be at the last of the pipeline as much as possible, so that it can prevent vulnerabilities if any other filters generate vulnerable result.